src/utils/spo.ts

```diff
@@ -183,6 +183,7 @@ export const spo = {
         .get<{ webUrl: string }>(requestOptions)
         .then((res: { webUrl: string }): Promise<void> => {
           auth.service.spoUrl = res.webUrl;
+          auth.updateAvailableIdentitiesList();
```

Comment:
Why do we need to update the list of identities here?

Reply:
Same reason as above...